### PR TITLE
Fix & enhance man page

### PR DESCRIPTION
**Pull request by tastytea**
- Fix documentation for `--profile`
- Document emoji pickers
- Add FAQ with the question “How do I add stickers and custom emojis?”

I get asked about image packs a lot and it crops up in chat now and then, so I thought it's a good idea to document it. 😊

### CODE REVIEW

File: man/nheko.1.adoc.in [hunk 1]
```diff
@@ -33,9 +33,31 @@ Displays version information.
...
-*-p*, *--profile*::
+*-p* _<profile>_, *--profile* _<profile>_::
 Creates a unique profile, which allows you to log into several accounts at the
-same time and start multiple instances of nheko.
+same time and start multiple instances of nheko. Use _default_ to start with the
+default profile.
+
+== FAQ
+
+=== How do I add stickers and custom emojis?
```
Comment (deepbluev7) [inline man/nheko.1.adoc.in:43]: Can you add that to the README too? ;p

Comment (tastytea) [reply man/nheko.1.adoc.in:43]: sure!

File: man/nheko.1.adoc.in [hunk 2]
```diff
@@ -147,6 +172,15 @@ Inserts `┯━┯╭( º _ º╭)`
 */sovietflip*::
 Inserts `ノ┬─┬ノ ︵ ( \\o°o)\\`
 
+=== Inline commands
+
+*:*::
+Open unicode emoji picker.
+
+*~*::
```
Comment (deepbluev7) [inline man/nheko.1.adoc.in:180]: Maybe you should mention @, # and <tab> for completeness sake then too, if you name the completers "inline commands" ;p

Comment (tastytea) [reply man/nheko.1.adoc.in:180]: Haha, yeah. I totally forgot. I'm open to suggestions for a better term, btw.. Shold I call them completers instead?

Comment (deepbluev7) [reply man/nheko.1.adoc.in:180]: Yeah, completers is the name I always use :3

**Comment by deepbluev7** [comment]
Thank you <3

File: man/nheko.1.adoc.in [hunk 3]
```diff
@@ -33,9 +33,49 @@ Displays version information.
...
+Emote Settings. You will see the Private pack (your user image pack) and can
+edit it there. The Shortcode is a unique identifier you use to select the image
+with and the Body is the alternate text that is displayed on clients that don't
+support images (like the alt tag in HTML images).
+
+If you have the necessary permissions you can create new room packs with the
+button at the bottom of the image pack list. They are the same as the user pack
+but you can create as many as you want and everyone in the room can use them. It
+might be a good idea to create an extra room just for your image packs to make
+sharing easier.
```
Comment (McSinyx) [inline man/nheko.1.adoc.in:60]: Sorry for the necro, but _sharing_ here refers to using the image pack across room by the same user, i.e. globally enable packs are not available to other users in other room, right?

Comment (tastytea) [reply man/nheko.1.adoc.in:60]: everyone who is in the same room as the pack can enable it globally. other people can't.

**Pull request merged**